package.json

```diff
@@ -18,5 +18,14 @@
   },
   "engines": {
     "node": "^16.18"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.5.2",
+    "@fortawesome/fontawesome-svg-core": "^6.5.2",
+    "@fortawesome/free-brands-svg-icons": "^6.5.2",
+    "@fortawesome/free-regular-svg-icons": "^6.5.2",
+    "@fortawesome/free-solid-svg-icons": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "react-fontawesome": "^1.7.1"
   }
 }
```

Comment:
uh oh! this is at the root of the app, but these specifically should be added to the client/package.json. We shouldnt ever need to add dependencies to the root one.